examples/ddi_seal/config.py

```diff
@@ -0,0 +1,59 @@
+from yacs.config import CfgNode as CN
+import os
+
+_C = CN()
+
+# -----------------------------------------------------------------------------
+# Basic
+# -----------------------------------------------------------------------------
+_C.OUTPUT_DIR = 'results'
+
+# -----------------------------------------------------------------------------
+# Dataset
+# -----------------------------------------------------------------------------
+_C.DATASET = CN()
+_C.DATASET.ROOT = "dataset/ogbl_ddi"
```

Comment:
Some instructions are needed in ReadMe.

Reply:
This is the default root for ogb dataset download. If the dataset is not on local, it'll be automatically downloaded in this root. I think it doesn't need to refer in ReadMe since there is no extra operation for dataset acquirement. I can add instruction in config.py directly.